### ISSUE TITLE
White Color not suitable 

### ISSUE THREAD

**Issue by sabikb** [FACG5/NextInTeam#22]
https://github.com/FACG5/NextInTeam/blob/9fae54b1cbb57d9ab755089906ebf6931b0aa30d/style.css#L175

I prefer to use black color or grey , to increase accessibility
